src/Plugins/RestServer/RestServer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "PluginConfiguration": {
+    "Network": 860833102,
+    "BindAddress": "127.0.0.1",
+    "Port": 10339,
+    "KeepAliveTimeout": 120,
+    "SslCertFile": "",
+    "SslCertPassword": "",
+    "TrustedAuthorities": [], // Example: "aec13cdd5ea6a3998aec14ac331ad96bedbb770f" (Thumbprint)
+    "EnableBasicAuthentication": false,
+    "RestUser": "",
+    "RestPass": "",
+    "EnableCors": true,
+    "AllowOrigins": [], // Example: "http://www.example.com"
```

Comment:
```suggestion
    "AllowOrigins": [],
```

Reply:
Previously we discussed about this and decided that we shall not add comment to the json config file, maybe add readme for it instead?